cli/src/commands/util.rs

```diff
@@ -244,6 +244,29 @@ impl ShellCompletion {
             Self::Zsh => generate(Shell::Zsh, cmd, bin_name, &mut buf),
         }
 
+        // Append custom completions. The main purpose of these is to support
+        // dynamic completions, like branch names. This may be removed in the
+        // future when clap has builtin support for dynamic completions, see:
+        // https://github.com/clap-rs/clap/issues/1232
+        match self {
+            Self::Bash => {}
+            Self::Elvish => {}
+            Self::Fish => buf.write_all(CUSTOM_FISH_COMPLETIONS).unwrap(),
+            Self::Nushell => {}
```

Comment:
Minor & optional: Perhaps just use `_ => {}` syntax?

Reply:
Oh yeah, I tried that at first. But clippy warns about a single match arm, suggesting`if let` instead. Adding an allow annotation would be possible too.

Reply:
I think `if let` is fine too. Or just leave it be (until clippy gets smart enough to demand converting this code to if let as well).